scripts/genKey.js

```diff
@@ -0,0 +1,16 @@
+require('dotenv').config()
+const db = require('../db')
+const logSymbols = require('log-symbols')
+const generatePassword = require("password-generator")
+
+const key = generatePassword(12,false)
```

Comment:
what does `false` stand for here?

Reply:
Adds numbers to the generated password and prevents them from being just letters.
https://www.npmjs.com/package/password-generator

Reply:
Great, I think I was looking at some different documentation. It's clear now :)